src/lib/load/document.ts

```diff
@@ -22,7 +25,7 @@ export default async function load({ fetch, params, url }: Load) {
 
   let mode: ViewerMode =
     (url.searchParams.get("mode") as ViewerMode) ?? "document";
-  const text = await documents.text(document, fetch);
+  const text = { updated: 0, pages: [] }; // await documents.text(document, fetch);
```

Comment:
It looks to me like we're no longer referencing `data.text` in any +page functions—can we remove this?